docs/releases/3.1.txt

```diff
@@ -501,6 +508,9 @@ backends.
   yields a cursor and automatically closes the cursor and connection upon
   exiting the ``with`` statement.
 
+* ``DatabaseClient.runshell()`` now requires an additional ``parameters``
+  argument for extra arguments to pass on to the command-line client.
```

Comment:
Just the one comment/question: `parameters` is a list right? Is it worth saying it's a list here?

Reply:
Rephrased